app/src/components/validators.js

```diff
@@ -0,0 +1,315 @@
+const bytes = require('bytes');
+const fs = require('fs');
+const log = require('npmlog');
+const tmp = require('tmp');
+const validator = require('validator');
+
+const DEFAULT_ATTACHMENT_SIZE = bytes.parse('5mb');
+
+const asyncForEach = async (array, callback) => {
+  for (let index = 0; index < array.length; index++) {
+    await callback(array[index], index, array);
```

Comment:
Please fix this code smell from Sonarqube - 3 arguments to a 2 argument function. ([Source](https://sonarqube-9f0fbe-tools.pathfinder.gov.bc.ca/project/issues?id=common-hosted-email-service-pr-12&open=AW1lB9UMb9Iu5LLx-i12&resolved=false&types=BUG))

Reply:
fixed.
will consider the build logs warnings later.